pythonAPI/enable_logging_api.py

```diff
@@ -0,0 +1,32 @@
+"""
+This module enables the logging API for a cloud project
+"""
+import json
+import requests
+def enable_loggin_apis_pvt(project_number, token):
+    """
+    Function for enabling the logging API for a cloud project
+
+    Parameters:
+        project_number (int):Project Number of the cloud project
+        token (str):The authorization token for cloud Project
+
+    Returns:
+        boolean: whether logging API is enabled or not
+    """
+    name = "projects/" + project_number + "/services/logging.googleapis.com"
+    url = "https://serviceusage.googleapis.com/v1/" + name + ":enable"
```

Comment:
Can you check what will happen if you try to enable the API which is already enabled?

Reply:
If it's enabled already then no change in state is there